HEC.FDA.ViewModel/AggregatedStageDamage/CalculatedStageDamageVM.cs

```diff
@@ -753,7 +754,17 @@ private void WriteDetailsCsvFile(ScenarioStageDamage scenarioStageDamage, List<U
         {
             try
             {
-                List<string> stageDamageDetails = scenarioStageDamage.ProduceStructureDetails();
+                //Its necessary for the details to be written with names for the impact areas, not FIDs. It helps users troubleshoot. I don't like this, but that's why the dictionary is passed in. to translate at the
+                //lowest level before it's written
```

Comment:
one day we'll write structure details to shapefile and it will be much cleaner than this

Reply:
writing structure details, anyway. 

your approach is clean, it makes sense